src/metabolic_ninja/worker/designer.py

```diff
@@ -284,6 +321,28 @@ def evaluate_cofactor_swap(designs, pathway, model, method):
                 "product": prod_flux,
                 "biomass": growth,
                 "method": method,
+                "targets": reaction_targets,
             }
         )
     return results
+
+
+def get_target_data(model, table, targets, knockout):
+    result = {}
+    for t in targets:
+        rxn_id = t.id if knockout else t["id"]
+        rxn = model.reactions.get_by_id(rxn_id)
+        result[rxn_id] = {
+            "name": rxn.name,
+            "subsystem": rxn.subsystem,
+            "gpr": rxn.gene_reaction_rule,
+            "definition_of_stoichiometry": rxn.build_reaction_string(True),
+            "flux_reversal": bool(
+                table[table.reaction == rxn_id].flux_reversal.values[0]
```

Comment:
You can make use of full [pandas indexing](https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html) here:
```suggestion
                table.at[rxn_id, "flux_reversal"]
```